static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -263,7 +265,7 @@ const submitForm = (
                 listId
             }&campaignCode=${formData.campaignCode}&referrer=${
```

Comment:
if we're really no longer using these in the backend then we can get rid of `campaignCode`, `referrer` and `listId`

Reply:
ListId is used in the GA reporting so might have to check if anyone is using that, also used in local storage to decide wether to show the widgets or not, maybe could use the identityListNames instead and get rid though yeah, the rest can hopefully go.